bin/acli

```diff
@@ -65,7 +65,7 @@ $kernel = new Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
 // Handle a cache:clear pseudo command. This isn't implemented as a true console
 // command because a stale or corrupted cache would render it unusable--
 // precisely when it is needed.
-if (in_array($input->getFirstArgument(), ['cache:clear', 'cc'])) {
+if (in_array($input->getFirstArgument(), ['clear-internal-cache', 'cci'])) {
   $filesystem = new Filesystem();
   $kernel_cache_dir = $kernel->getCacheDir();
   $system_cache_dir = Path::join(sys_get_temp_dir(), 'symphony-cache');
```

Comment:
I think we'd want to clear the "system cache" via the new clear-caches command. It's in a temporary system directory so it should be good/safe to clear.

I think we should rename this pseudo-command to `clear-kernel-cache` or something, and have it only clear the kernel cache (that lives in `app-dir/var`).

There's also a comment above this conditional that should be renamed.